src/js/profile/mobile/config.js

```diff
@@ -34,6 +34,7 @@
 			ns.setConfig("popupTransition", "slideup", true);
 			ns.setConfig("pageTransition", "slide", true);
 			ns.setConfig("enablePageScroll", false, true);
+			ns.setConfig("keyboardSupport", false);
```

Comment:
This line is not needed. The common config exists in: src/js/core/config.js. KeyboradSupport is disabled by default.

Reply:
Done. Thanks.